src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Color;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COUNT_OF_FIGURES = 5;
+    private static final double MAX_LENGTH = 25.0;
+    private static final double MIN_LENGTH = 1.0;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
```

Comment:
pretty much the same remark as in the [comment](https://github.com/mate-academy/jv-oop-advanced/pull/1220/files#r1291232497) :)